src/get-started/beam-overview.md

```diff
@@ -36,6 +36,8 @@ Beam currently supports Runners that work with the following distributed process
          alt="Apache Flink">
 * Apache Gearpump (incubating) <img src="{{ site.baseurl }}/images/logos/runners/gearpump.png"
          alt="Apache Gearpump">
+* Apache Hadoop MapReduce <img src="{{ site.baseurl }}/images/logos/runners/mapreduce.png"
```

Comment:
this image is very big compared to the others on the page... may need to specify dimensions or use a smaller image. more a comment for later, as from Kenn's comment, it sounds like this runner shoudn't be listed here yet.

Reply:
done